src/components/Article/BottomMenu/DocPad/DocPadEditor/Toolbar/index.tsx

```
@@ -0,0 +1,243 @@
+        <AlignCenter className={iconStyles({ textAlign: 'center' })} />
+      </ToolbarButton>
+      <ToolbarButton
+        onClick={() => editor.chain().focus().setTextAlign('right').run()}
+        description="Set Align right"
+      >
+        {/* Allign Right */}
+        <AlignRight className={iconStyles({ textAlign: 'right' })} />
+      </ToolbarButton>
+
+      <ToolbarButton
+        onClick={() => editor && editor.chain().focus().toggleBold().run()}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)